### ISSUE TITLE
Can't get Go to Definition on Character/isWhitespace

### ISSUE THREAD

**Issue by brettrowberry** [BetterThanTomorrow/calva#1384]
`Go to Definition` seems to work most of the time, but not in cases like this:
1. Open a Clojure file containing `clojure.string.blank?`
2. Run `Go to Definition` on the call to `clojure.string.blank?`
3. Run `Go to Definition` on the call to `Character/isWhitespace`
4. `No definition found for 'Character/isWhitespace'`

I don't know if the problem is that `Character/isWhitespace` is defined in Java or some other reason.

I'm curious to know if there is a resolution - whether that be another plugin, setting, anything! Thanks!

**Comment by PEZ** [comment]
Hello, thanks for reaching out!

This could be that the Java sources are not found on the classpath/where cider-nrepl is looking for them. We have yet to figure out the correct instructions for this: https://github.com/BetterThanTomorrow/calva/issues/1063

**Comment by brettrowberry** [comment]
Wait, Calva is working just fine!

VS Code 1.62.2
Calva 2.0.225
Clojure CLI 1.10.3.943
openjdk 11.0.12 2021-07-20

I made a 
- `deps.edn` containing `{}`
- `src/hello.clj` containing
```clojure
(ns hello
  (:require [clojure.string :as string]))

(string/blank? "hello")
```

I jacked in, evaluated the `ns` form, went to definition on `blank?`, went to definition on `isWhitespace`.
It opened `Character.java`!

The problem may just be with our monorepo's classpath stuff.
